src/marqo/tensor_search/enums.py

```diff
@@ -153,10 +153,10 @@ class HealthStatuses(str, Enum):
     yellow = "yellow"
     red = "red"
 
-    def __lt__(self, other):
+    def __gt__(self, other):
         # Define the custom comparison logic
         status_order = [self.green, self.yellow, self.red]
-        return status_order.index(self) < status_order.index(other)
+        return status_order.index(self) > status_order.index(other)
```

Comment:
Are we 100% using __gt__ rather than __lt__ won't break @wanliAlex 's changes? Otherwise we can just define both?

Reply:
defined both!